src/lib/utils.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+
+const autoGenerateProductPayload = () => {
```

Comment:
this snippet looks like from different PR

Reply:
yes, they have the same commits as the other .... once the other is merged these commits will not be shown here